src/localization/locales/en-US.json

```diff
@@ -538,7 +538,33 @@
         "map_create_document_title": "Create Map: {{name}}",
         "map_view_document_title": "View Map: {{name}}",
         "fetchVisualizationConfig.not_found": "Map not found",
-        "addVisualizationConfig.unique": "The name “{{body.variables.input.title}}“ is in use by another map. Please use a different name."
+        "addVisualizationConfig.unique": "The name “{{body.variables.input.title}}“ is in use by another map. Please use a different name.",
+        "tab_files_zero": "Upload files",
+        "tab_files": "Upload files ({{count}})",
+        "tab_links_zero": "Share links",
+        "tab_links": "Share links ({{count}})",
+        "upload_summary_files_one": "1 file",
+        "upload_summary_files": "{{count}} files",
+        "upload_summary_links_one": "1 link",
+        "upload_summary_links": "{{count}} links",
+        "upload_summary_separator": " and ",
+        "upload_summary_success_zero": "",
+        "upload_summary_success_one": "{{successCounts}} has been shared successfully.",
+        "upload_summary_success": "{{successCounts}} have been shared successfully.",
+        "upload_summary_error_zero": "",
+        "upload_summary_error": "{{errorCounts}} could not be shared. ",
+        "upload_summary_zero": "",
+        "upload_summary": "[TBD] $t(sharedData.upload_summary_success, {\"count\": {{successCount}} })$t(sharedData.upload_summary_error, {\"count\": {{errorCount}} })Fix the issues and share again or skip sharing files or links with issues.",
+        "upload_url_label": "Web Address",
+        "upload_name_label": "Name",
+        "upload_links_add": "Add link",
+        "upload_links_description": "Add links and descriptions.",
+        "upload_link_success": "This link has been shared successfully.",
+        "upload_link_no_name": "Link {{index}}",
+        "upload_link_form.url": "$t(sharedData.upload_url_label)",
+        "upload_link_form.name": "$t(sharedData.upload_name_label)",
+        "addSharedDataLink.terraso_api.invalid": "Invalid $t(sharedData.upload_{{field}}_label)",
+        "link_delete_tooltip": "Remove {{name}} link from the shared files and links."
```

Comment:
```suggestion
        "link_delete_tooltip": "Remove the link “{{name}}”."
```

Reply:
question: is there a different tooltip for deleting files?

Reply:
@paulschreiber No, there is no tooltip for deleting files

Reply:
@paulschreiber The intent of this tooltip (from the design) is to let the user know explicitely that this will not delete the source of truth, it will just remove it from this section in terraso, please sync with Shivani about this change